src/components/input/demoBasicUsage/index.html

```diff
@@ -53,7 +53,7 @@
         </md-input-container>
         <md-input-container flex>
           <label>Postal Code</label>
-          <input ng-model="user.postalCode">
+          <input ng-model="user.postalCode" placeholder="12345">
```

Comment:
This uses the `placeholder` directive. But what uses/inserts the `md-placeholder` style?